packages/create/templates/hello-v2/dbos-config.yaml

```diff
@@ -0,0 +1,20 @@
+# To enable auto-completion and validation for this file in VSCode, install the RedHat YAML extension
+# https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml
+
+# yaml-language-server: $schema=https://raw.githubusercontent.com/dbos-inc/dbos-transact/main/dbos-config.schema.json
+
+language: node
+database:
+  hostname: localhost
+  port: 5432
+  username: postgres
+  password: ${PGPASSWORD}
+  connectionTimeoutMillis: 3000
+  app_db_client: knex
+  migrate:
+    - npx knex migrate:latest
+  rollback:
+    - npx knex migrate:rollback
+runtimeConfig:
+  entrypoints:
```

Comment:
Same as above, should we change it to use the new start command?

Reply:
Not for this one, it still uses serverless entrypoints.
If you compare, the serverless version gives you a handful of features for free with less lines of code since you don't set up the server.